Fixed the celery PVC name in cronjob

zagganas opened the pull request:
**Details**

In my previous pull request there was an error in the celery tmp PVC name in the cronjob definition. This PR fixes that.

zagganas committed:
Bumped ingress to new specs for k8s 1.22

lvarin reviewed (dismissed):
LGTM

I don't know how I did not see the problem in the previous PR?

lvarin commented:
I was about to merge this but saw that there are 11 commits to merge, and most of this are merge commits that make the history complicarted. This can be solved by rebasing. Could you please rebase your dev branch?

lvarin commented:
Or I can just squish merge ￼  the PR

zagganas committed:
parent bcacf04d8ca314ce425394a33be52206f58dcc19
author zagganas <zagganas@athenarc.gr> 1637319108 +0000
committer zagganas <zagganas@athenarc.gr> 1637321534 +0000

parent bcacf04d8ca314ce425394a33be52206f58dcc19
author zagganas <zagganas@athenarc.gr> 1637319108 +0000
committer zagganas <zagganas@athenarc.gr> 1637321273 +0000

Fixed celery /tmp PVC in Cronjob

build: remove conflicting dependency (#218)

Co-authored-by: Alex Kanitz <alexander.kanitz@alumni.ethz.ch>

fix: accept hyphens in task names (#188)

build: redirect base route to Swagger UI (#212)

fix: retain configmap upon Helm upgrade (#215)

build(deps): bump cryptography from 3.2 to 3.3.2 (#210)

Bumps [cryptography](https://github.com/pyca/cryptography) from 3.2 to 3.3.2.
- [Release notes](https://github.com/pyca/cryptography/releases)
- [Changelog](https://github.com/pyca/cryptography/blob/master/CHANGELOG.rst)
- [Commits](https://github.com/pyca/cryptography/compare/3.2...3.3.2)

Signed-off-by: dependabot[bot] <support@github.com>

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

fix: update cwl-tes log parsing (#209)

The build fails because the image mher/flower:0.9 in docker-compose.yaml does not exist (anymore?). But mher/flower:0.9.7 exists. (#224)

Fixed api version of CronJob for WES ingress

fix: rename supported file systems property (#226)

Added an if/else clause for cronjob api version

Bumped flask cores from 3.0.6 to 3.0.9

bump Flask-Cors to 3.0.9 (#227)

* Bumped flask cores from 3.0.6 to 3.0.9

Fixed celery pvc name in cronjob

zagganas committed:
Merge branch 'dev' into dev
